packages/pn-commons/src/pages/AccessDenied.tsx

```diff
@@ -16,6 +17,7 @@ const AccessDenied: React.FC<Props> = ({
   goToHomePage,
   message,
   subtitle,
+  qrError = false
```

Comment:
This is a common component, so i wouldn't use a so specific property.
I would change this property with a more generic "icon" property. This property will be of type Illustration (check the right type).
So IllusQuestion will not used here but passed from outside